doc/shell_long/README.md

```diff
@@ -0,0 +1,635 @@
+./1hour/bug_bts_15629/cases/bug_bts_15629.sh
+./1hour/bug_bts_15419_1_asc/cases/bug_bts_15419_1_asc.sh
+./1hour/bug_bts_5222/cases/bug_bts_5222.sh
+./1hour/bug_bts_17881/cases/bug_bts_17881.sh
+./1hour/bug_bts_18031/cases/bug_bts_18031.sh
+./1hour/bug_bts_9382/cases/bug_bts_9382.sh
+./1hour/bug_bts_15419_1_desc/cases/bug_bts_15419_1_desc.sh
+./1hour/bug_bts_15419_2_asc/cases/bug_bts_15419_2_asc.sh
+./1hour/bug_bts_7350/cases/bug_bts_7350.sh
+./1hour/_02_cursor_stress/cases/_02_cursor_stress.sh
+...
+./2hour/_02_cursor_stress/cases/_02_cursor_stress.sh
+./2hour/bug_bts_10009/cases/bug_bts_10009.sh
+./2hour/bug_bts_9969/cases/bug_bts_9969.sh
+```
+How to write new shell_long test cases   
+please see [shell long cases specification](#5test-case-specification)
+
+# 3. Deploy Shell_long Regression Test
+In this section, we will introduce how to deploy the Shell_long Regression Test Environment.  
+## 3.1 Test Machines
+For shell_long regression test, we usually have one controller node and multiple test nodes.In our regression configurations,we use five test nodes.  
+**Controller node** : This node listens to test messages and starts a test when there is a test message.  
+**Test node** : CUBRID server is deployed on this node,we execute test cases on it. 
+
+Information about each test machines.
+<table>
+<tr>
+<th>Description</th>
+<th>User Name</th>
+<th>IP</th>
+<th>Hostname</th>
+<th>Tools to deploy</th>
+</tr>
+<tr class="even">
+<td>controller node</td>
+<td>rqgcontroller</td>
+<td>192.168.1.99</td>
+<td>func24</td>
+<td><p>CTP</p>
+</td>
+</tr>
+<tr class="even">
+<td>test node</td>
+<td>shell_long</td>
+<td>192.168.1.99</td>
+<td>func24</td>
+<td><p>CTP</p>
+<p>cubrid-testcases-private</p>
+</td>
+</tr>
+<tr class="even">
+<td>test node</td>
+<td>shell_long</td>
+<td>192.168.1.100</td>
+<td>func25</td>
+<td><p>CTP</p>
+<p>cubrid-testcases-private</p>
+</td>
+</tr>
+<tr class="even">
+<td>test node</td>
+<td>shell_long</td>
+<td>192.168.1.101</td>
+<td>func26</td>
+<td><p>CTP</p>
+<p>cubrid-testcases-private</p>
+</td>
+</tr>
+<tr class="even">
+<td>test node</td>
+<td>shell_long</td>
+<td>192.168.1.102</td>
+<td>func27</td>
+<td><p>CTP</p>
+<p>cubrid-testcases-private</p>
+</td>
+</tr>
+<tr class="even">
+<td>test node</td>
+<td>shell_long</td>
+<td>192.168.1.103</td>
+<td>func28</td>
+<td><p>CTP</p>
+<p>cubrid-testcases-private</p>
+</td>
+</tr>
+</table>
+
+## 3.2	Test Deployments
+### For all nodes
+1. Install JDK
+2. Add IP and hostname to /etc/hosts
+```
+192.168.1.99  func24  
+192.168.1.100  func25    
+192.168.1.101  func26  
+192.168.1.102  func27  
+192.168.1.103  func28   
+```
+3. checkout git repositories to $HOME/
+```
+git clone https://github.com/CUBRID/cubrid-testtools.git
+cd ~/cubrid-testtools/
+git checkout develop
+```
+### On Controller node  
+1. Install CTP      
+Please follow [the guide to install CTP](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md).

```

Comment:
Change link to https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform